### ISSUE TITLE
Update IPFS tests

### ISSUE THREAD

**Issue by srirambv** [brave/qa-resources#512]
Update IPFS Tests

Items to be updated

* Update IPFS installation section to reflect Kubo instead of go-ipfs
* URL Updates
Ref: https://github.com/brave/brave-browser/issues/22927
* Gateway URL's
* IPFS Companion - remove section for `Set your IPFS preference`

**Comment by kjozwiak** [comment]
@srirambv still valid?

**Comment by srirambv** [comment]
Yes still valid. Haven't had time to work on this. Will get this done

**Comment by kjozwiak** [comment]
Closing as the above isn't valid anymore as we've deprecated IPFS from `Brave` due to low usage numbers.
